coil/src/main/java/dev/chrisbanes/accompanist/coil/Coil.kt

```diff
@@ -179,34 +163,49 @@ fun CoilImage(
         // We remember the last size in a MutableRef (below) rather than a MutableState.
         // This is because we don't need value changes to trigger a re-composition, we are only
         // using it to store the last value.
-        val lastRequestedSize = remember(requestActor) { MutableRef(IntSize.Zero) }
+        val lastRequestedSize = remember(requestActor) { MutableRef<IntSize?>(null) }
```

Comment:
Use delegate?

Reply:
Can't, this is a custom `MutableRef` rather than `MutableState` (so no `getValue`, etc)